test/utils/safes.js

```diff
@@ -0,0 +1,58 @@
+import Safe from '@circles/safe-contracts/build/contracts/GnosisSafe.json';
+import ProxyFactory from '@circles/safe-contracts/build/contracts/ProxyFactory.json';
+
+import web3 from './web3';
+
+import { ZERO_ADDRESS } from '~/constants';
+
+// Helper function used in tests to create Safe instances associated to the
+// given ganache accounts. The account 0 is used as the admin account
+// and it doesn't have a Safe.
+export async function createSafes(accounts) {
```

Comment:
This looks really cool! I suggest having two method arguments, one for the funder account and the others for an array of the actual accounts to be added to Safes.

Reply:
Done ! 😺